test/geometry/test_plane.py

```diff
@@ -1,3 +1,5 @@
+import tempfile
```

Comment:
We should avoid this in tests in favor of `tmp_path` / `tmp_path_factory` from pytest